From b5c40f346c73871628983afc4a2d27a120196e7c Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 17 Apr 2024 11:18:21 +0100
Subject: [PATCH 1027/1085] DTS: rp1: fix setting xHCI TX burst fifo thresholds

The property should be a u8, and should target the non-periodic
SuperSpeed transmit FIFO not the periodic one.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/rp1.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm/boot/dts/broadcom/rp1.dtsi
+++ b/arch/arm/boot/dts/broadcom/rp1.dtsi
@@ -1077,8 +1077,8 @@
 			snps,parkmode-disable-ss-quirk;
 			snps,parkmode-disable-hs-quirk;
 			snps,parkmode-disable-fsls-quirk;
-			snps,tx-max-burst-prd = <8>;
-			snps,tx-thr-num-pkt-prd = <2>;
+			snps,tx-max-burst = /bits/ 8 <8>;
+			snps,tx-thr-num-pkt = /bits/ 8 <2>;
 			interrupts = <RP1_INT_USBHOST0_0 IRQ_TYPE_EDGE_RISING>;
 			status = "disabled";
 		};
@@ -1093,8 +1093,8 @@
 			snps,parkmode-disable-ss-quirk;
 			snps,parkmode-disable-hs-quirk;
 			snps,parkmode-disable-fsls-quirk;
-			snps,tx-max-burst-prd = <8>;
-			snps,tx-thr-num-pkt-prd = <2>;
+			snps,tx-max-burst = /bits/ 8 <8>;
+			snps,tx-thr-num-pkt = /bits/ 8 <2>;
 			interrupts = <RP1_INT_USBHOST1_0 IRQ_TYPE_EDGE_RISING>;
 			status = "disabled";
 		};
